Plotter/python/sample/utils.py

```diff
@@ -14,7 +14,7 @@
 LOG  = Logger('Sample')
 era  = None # data period: 2016, 2017, 2018, ...
 lumi = -1   # integrated luminosity [fb-1]
-cme  = 13   # center-of-mass energy [TeV]
+cme  = 13.6   # center-of-mass energy [TeV]
```

Comment:
Hi @danielwinterbottom, I know this is already merged, but it's a bit dangerous to change the defaults here for users who are still using Run-2 with CME 13 TeV.

For backward compatibility, it would be better if we overwrite the default CME in the scripts that make Run 3 samples & plots by passing `cme=13.6` to the relevant functions like `setera`, `SampleSet.stitch`, etc.

Or the global default can be set with something like
```
import TauFW.Plotter.sample.utils
TauFW.Plotter.sample.utils.cme = 13.7
```
and then the defaults in `setera` and `stitch` should become
```
cme_ = kwargs.get('cme',cme)
```
where `cme_` is a new local variable.